INSTALL.md

```diff
@@ -1,3 +1,5 @@
+TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker